main.js

```diff
@@ -7,3 +7,69 @@ const close = document.querySelector('#slide_menu');
 close.addEventListener('click', () => {
   document.getElementById('slide_menu').classList.remove('active');
 });
+
+const projects = [
+  {
+    name: 'Tonic',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    image: './images/tonic.svg',
+    technologies: ['html', 'css', 'JavaScript'],
+    link: 'https://www.tripadvisor.co/Attractions-g652402-Activities-Manizales_Caldas_Department.html',
+    source: 'https://github.com/robinbenitezmora/Mobil-Version',
+  },
+  {
+    name: 'Multi-Post Stories',
+    description: 'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
+    image: './images/big-image4.png',
+    technologies: ['html', 'Ruby on rails', 'css', 'JavaScript'],
+    link: 'https://www.tripadvisor.co/Attractions-g652402-Activities-Manizales_Caldas_Department.html',
+    source: 'https://github.com/robinbenitezmora/Mobil-Version',
+  },
+  {
+    name: 'Facebook 360',
+    description: 'Exploring the future of media in Facebook first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.',
+    image: './images/photo-portfolio.svg',
+    technologies: ['html', 'Ruby on rails', 'css', 'JavaScript'],
+    link: 'https://www.tripadvisor.co/Attractions-g652402-Activities-Manizales_Caldas_Department.html',
+    source: 'https://github.com/robinbenitezmora/Mobil-Version',
+  },
```

Comment:
- [ ] Good job working on the changes requested by the previous reviewer, all the "see project" buttons are working. However, the pop-up design no longer looks like the template after the changes were made. Kindly fix it.
Your design
![Capture](https://user-images.githubusercontent.com/85095702/169653227-b735cc53-1b4e-4b2b-841a-a049fdec1ed7.PNG)

Figma template
![figma](https://user-images.githubusercontent.com/85095702/169653259-404fe05d-0470-4ed7-96e5-646eee5e13a7.PNG)

Reply:
- [ ] This issue is not yet resolve :heavy_multiplication_x:

Reply:
- [ ] This issue is not yet resolved :x:

Reply:
**[REQUIRED CHANGES]**

- As required by the previous reviewer, please kindly resolve the issue with your modal to match the figma design as stated above